python/pynessie/cli.py

```diff
@@ -46,10 +46,22 @@
 class ContextObject(object):
     """Click context object."""
 
-    nessie: NessieClient
+    config: confuse.Configuration
     verbose: bool
     json: bool
 
+    __nessie: Optional[NessieClient] = None
```

Comment:
It's private ;)

Reply:
its still static here. Not sure why

Reply:
Python's a mystery ...

Reply:
Its just not Java. I am at fault for calling an instance variable static, which it isn't in the Java sense.  This may clarify things: https://dev.to/ogwurujohnson/distinguishing-instance-variables-from-class-variables-in-python-81